tools/distribution/release.py

```diff
@@ -73,9 +72,6 @@
         if only_cocoapods and overwrite_github:
             raise Exception('`--overwrite-github` and `--only-cocoapods` cannot be used together.')
 
-        if only_cocoapods and add_xcode_version_to_github_asset:
```

Comment:
![image](https://github.com/DataDog/dd-sdk-ios/assets/8882380/a0e09ae8-d8cf-498a-bb1f-f1265ac3cbe0)
can we remove ^^ also?

Reply:
good catch!